_episodes/05-preprocessor.md

```diff
@@ -0,0 +1,287 @@
+---
+title: "Working with preprocessors"
+teaching: 20?
+exercises: 15?
+questions:
```

Comment:
Maybe also add the question: "How can I combine different preprocessor functions?"

Reply:
Done.